.github/workflows/push.yml

```diff
@@ -0,0 +1,25 @@
+name: Push
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set Up Linux
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libv4l-dev libudev-dev libasound2-dev
+    - uses: actions/checkout@v1
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
```

Comment:
actions-rs is unmaintained, and support for it will be dropped next month. Can we switch this out for either https://github.com/dtolnay/rust-toolchain or https://github.com/actions-rust-lang/setup-rust-toolchain ?